website/docs/docs/core-versions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Understanding dbt Core versions"
```

Comment:
Should this just be `Understanding dbt versions`, or `dbt Core versions`? I see it's the widest thing on the default sidebar

Reply:
@jtcohen6 This is a great point. Maybe "About dbt Core versions?" I will be making those other titles more screen-reader friendly so this won't be the only longish one.